### DISCUSSION TITLE
tailwind 쉽게 짜는 법

### DISCUSSION THREAD

**Discussion by ho991217** [EveryUniv/next-student-council-frontend#13] [일반]
[여기](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss)에서 vscode의 tailwind extension을 활용하면 편합니다!

<img width="607" alt="스크린샷 2023-09-15 10 33 44" src="https://github.com/EveryUniv/next-student-council-frontend/assets/84632077/00948fb9-eb15-42ed-a38e-322815917b39">

이렇게 자동완성을 해 주기도하고

<img width="420" alt="스크린샷 2023-09-15 10 33 55" src="https://github.com/EveryUniv/next-student-council-frontend/assets/84632077/51d06fe0-2db4-40ba-b9e1-4e8b1ef0a6a5">

이렇게 마우스 호버하면 미리보기도 가능합니다~

**Comment by sy-paik** [comment]
오호.. 아직 익숙하지 않아서 공식문서 보면서 하는데 익스텐션 이용하면 더 익히는데 좋을 것 같네요! 참고하겠습니다~
